CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to the Cloud Azure Ops Ansible Collection
+ 
+We welcome community contributions to this collection.
+This guide provides clear instructions for contributors on how to set up their environment for contributing to the `cloud.azure_ops` Ansible Collection.
+If you find problems, please open an issue or create a Pull Request (PR) against this collection repository.
+
+## Prerequisites
+
+Make sure you have the following installed on your Machine:
+
+1. Python3 and pip3
+2. Docker or Podman  
+For more information on how containers are used with Ansible, refer to the [Containers section in the Ansible Community Documentation](https://docs.ansible.com/ansible/latest/dev_guide/testing_running_locally.html#containers).  
+To allow managing Docker as a non-root user, refer to the [Linux post-installation steps for Docker Engine](https://docs.docker.com/engine/install/linux-postinstall/).
+3. Ansible
+   This can be installed by running `pip3 install ansible`
+
+## Environment Setup
+
+To set up your environment for contributing to the Cloud Azure Ops Ansible Collection, follow these steps:
+
+> **_NOTE:_** Make sure to replace `<working directory>` with the actual path where you want to clone the repositories and set up your environment. 
+
+1. **Clone This Repository**:
```

Comment:
```suggestion
1. **Clone Azure Roles Collection Repository**:
```

Reply:
I changed the descriptions in the clone steps.
Please review